index.fr.md

```diff
@@ -0,0 +1,226 @@
+
+Les problèmes d'accessibilité peuvent être causés par un ou plusieurs composants. Par exemple, si un utilisateur qui ne peut pas utiliser une souris n'arrive pas à accéder aux fonctionnalités clavier des technologies d'assistance, il peut y avoir plusieurs explications :
+
+-   le développeur n'a pas balisé/codé correctement la page Web, ou
+-   le navigateur ou le lecteur multimédia ne lit pas correctement le code, ou
+-   la technologie d'assistance ne lit pas correctement le code, ou
+-   l'utilisateur ne sait pas comment utiliser le navigateur ou le lecteur multimédia ou les fonctionnalités accessibles au clavier des technologies d'assistance, ou
+-   la page n'est pas correment conçue et il s'agit d'un problème général d'utilisation pour tous les utilisateurs, y compris ceux qui ne se trouvent *pas* en situation de handicap.
+s
+## Associez l'évaluation des utilisateurs aux standards {#stdstoo}
+
+Inclure les utilisateurs handicapés dans l'évaluation présente de nombreux avantages. Ceci dit, cela ne suffit pas à déterminer l'accessibilité d'un site Web. Associez l'inclusion des utilisateurs à l'[évaluation de la conformité aux WCAG](/test-evaluate/conformance/) afin de garantir l'accessibilité pour différents types de handicaps et de situations.
+
+## Tirez des conclusions et créez un rapport {#drawing}
+
+Faites attention si vous tirez des conclusions d'évaluations ou d'études limitées. Les résultats obtenus avec deux personnes handicapées ne peuvent être généralisés et appliqués à toutes les personnes dans la même situation de handicap ou présentant d'autres handicaps. Retrouvez plus d'informations dans la section [Attention](#caution) ci-dessous.
+
+Incluez dans votre rapport le cadre de votre étude et les paramètres de l'évaluation, tels que les méthodes de test et les caractéristiques des utilisateurs. Par exemple, si une étude a inclu uniquement des personnes aveugles, le rapport devra préciser que celle-ci n'a pas évalué la conformité aux standards d'accessibilité et qu'elle ne s'applique pas à toutes les personnes handicapées. Bien que les études de petite envergure peuvent souvent fournir des informations utiles, elles ne sont pas assez solides pour fournir une signification statistique.
+
+## Note à l'attention des ergonomes {#ut-access}
+
+Lors de la recherche d'obstacles à l'accessibilité spécifiques, le protocole est généralement différent d'un test d'utilisation général. Par exemple :
+
+-   vous utiliseriez certainement un système de réfléxion à voix haute et une forte interaction avec l'organisateur des tests
+-   la collecte des données serait axée sur la compréhension des erreurs relatives aux obstacles à l'accessibilité, plutôt que sur le temps passé à réaliser une tâche ou sur la satifaction utilisateur
+-   les tâches seraient axées sur des parties spécifiques qui pourraient potentiellement poser problème, plutôt que sur l'utilisation générale du site
```

Comment:
To avoid repetition of "seraient axées" : "les tâches se concentreraient sur des zones spécifiques" ?